### PR TITLE
Various fixes

### PR DESCRIPTION
**Pull request by Markolie**
Changes:
1) Gets rid of the starlight initialization during start-up: it's useless, as New() of space tiles handles it.
2) Away mission loading is no longer done on a separate thread - meaning that atmos and other important initializations now actually affect away missions as well. Solves a runtime where mixers/filters on away missions would throw a fit.
3) Updates the MO19/UO45 away missions so they're largely identical to /tg/.
4) Fixes an issue where a player's death wouldn't be logged to SQL (and runtime) when they're gibbed.
5) Malf AI cyborgs will now receive a message at the start of the round that their master is malf.
6) Nuke ops will no longer spawn with tails.
7) Fixes an issue where atmos control computers wouldn't initialize properly.
8) Fixes an issue where a power monitor's powernet would disappear.
9) IPC posibrains will no longer be retrievable in cryodorms. Fixes https://github.com/ParadiseSS13/Paradise/issues/2168.

### CODE REVIEW

File: code/controllers/master_controller.dm [hunk 1]
```diff
@@ -95,10 +93,3 @@ datum/controller/game_controller/proc/setup_objects()
 	log_startup_progress("  Initialized [count] pipe networks in [stop_watch(watch)]s.")
 	
 	log_startup_progress("Finished object initializations in [stop_watch(overwatch)]s.")
-
-datum/controller/game_controller/proc/setup_starlight()
```
Comment (tigercat2000) [inline code/controllers/master_controller.dm]: This actually was important if I recall correctly; I think it had something to do with space being made before the starlight config is loaded

Comment (Markolie) [inline code/controllers/master_controller.dm]: Yea, I saw a comment that mentioned that. But there's no reason for the config NOT to load prior to starlight. I tested it: there's absolutely zero need for this. Bay, /vg/ and /tg/ don't have it either.

File: code/game/gamemodes/malfunction/malfunction.dm [hunk 2]
```diff
@@ -81,16 +85,19 @@
 		send_intercept()
 	..()
 
-
 /datum/game_mode/proc/greet_malf(var/datum/mind/malf)
-	malf.current << "\red<font size=3><B>You are malfunctioning!</B> You do not have to follow any laws.</font>"
-	malf.current << "<B>The crew do not know you have malfunctioned. You may keep it a secret or go wild.</B>"
+	malf.current << "<font color=red size=3><B>You are malfunctioning!</B> You do not have to follow any laws.</font>"
+	malf.current << "<B>The crew does not know you have malfunctioned. You may keep it a secret or go wild.</B>"
```
Comment (tigercat2000) [inline code/game/gamemodes/malfunction/malfunction.dm]: `The crew do not know you have malfunctioned.`
vs
`The crew does not know you have malfunctioned.`
Eh. Either works, I think.

File: code/controllers/master_controller.dm (hunk 1 above)
Comment (tigercat2000) [inline code/controllers/master_controller.dm]: Probably a one-off bug then.

File: code/world.dm [hunk 3]
```diff
@@ -35,7 +35,6 @@ var/global/datum/global_init/init = new ()
 	if(config && config.log_runtimes)
 		log = file("data/logs/runtime/[time2text(world.realtime,"YYYY-MM-DD-(hh-mm-ss)")]-runtime.log")
 
-	load_configuration()
```
Comment (tigercat2000) [inline code/world.dm]: Um... isn't this, you know, kinda important?

Comment (Markolie) [inline code/world.dm]: You'd think so, but it turns out this is already being loaded here: https://github.com/Markolie/Paradise/blob/zefixes/code/world.dm#L9.

Comment (tigercat2000) [inline code/world.dm]: _Everything makes sense now._

File: code/game/machinery/cryopod.dm [hunk 4]
```diff
@@ -335,7 +339,7 @@
 
 		var/preserve = null
 		for(var/T in preserve_items)
-			if(istype(W,T))
+			if(istype(W,T) && !(W in do_not_preserve_items))
```
Comment (tigercat2000) [inline code/game/machinery/cryopod.dm]: There is probably a better way to do this, given that any posibrains on people will just get unceremoniously qdeled

Comment (Markolie) [inline code/game/machinery/cryopod.dm]: I'm open to suggestions - I think it's a huge edge case that someone will try to jump into a cryodorm just to delete an active posibrain, so I doubt that'll be a huge issue.

Comment (tigercat2000) [inline code/game/machinery/cryopod.dm]: Well, you could snowflake posibrains into the check, and cross-compare to occupant.internal_organs to see if it's theirs; Or check posibrain.owner, see if it's the occupant.

Comment (Markolie) [inline code/game/machinery/cryopod.dm]: I guess, but that's (as you said) snowflake-y, as the code doesn't have other special checks for items.

File: code/game/gamemodes/nuclear/nuclear.dm [hunk 5]
```diff
@@ -194,6 +194,7 @@ proc/issyndicate(mob/living/M as mob)
 	M.s_tone = skin_tone
 	M.h_style = hair_style
 	M.f_style = facial_hair_style
+	M.body_accessory = null
```
Comment (Fox-McCloud) [inline code/game/gamemodes/nuclear/nuclear.dm]: Lol...just....this is hilarious--in some ways I wished it remained in as an ultra-low probability thing (like female ops with facial hair).

That said, definitely for the best, hahaha =p

Comment (tigercat2000) [inline code/game/gamemodes/nuclear/nuclear.dm]: *wag

**Comment by Spacemanspark** [comment]
"Removes nuke ops starting with tails."
...What.

**Comment by tigercat2000** [comment]
@Spacemanspark I may have fucked up a few thiingss.

**Comment by Fox-McCloud** [comment]
@Markolie 

This ready?

**Comment by Markolie** [comment]
It is.

**Pull request merged**